sip/nordic/sip.yml

```diff
@@ -0,0 +1,3 @@
+sips:
+  - name: nrf9161
+    soc: nrf9120
```

Comment:
this should be part of the soc.yml file so it's clearly hierarchical though would probably need a rename of the file

Reply:
I disagree, just like a board, the sip contains one or more socs, from one or more vendors. The soc is entirely independent of the sip, that's why the sip is defined seperately.

Consider a SiP from vendor `bar`, containing one soc from vendor `foo` and another soc from vendor `baz`, where the sip be defined?

With the schema in this PR, it would be defined in `sip/bar/sip.yml`:
```
sips:
  - name: bar_sip
    socs:
      - foo_soc
      - baz_soc
```

and any SiP specific overlays etc would go in `sip/bar/bar_sip/`.

It is clearly hierarchical, boards contain sips and socs, sips contain socs,

Reply:
right you are